### PR TITLE
Fixed ambiguous logging in DIETClassifier

### PR DESCRIPTION
**Pull request by tttthomasssss**
**Proposed changes**:
- Improved logging as it was unclear whether a log message comes from DIETClassifier or a subclass.

**Changes made**:
- added class name of caller to log message

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: rasa/nlu/classifiers/diet_classifier.py [hunk 1]
```diff
@@ -267,7 +267,7 @@ def __init__(
 
         if component_config is not None and EPOCHS not in component_config:
             logger.warning(
-                f"Please configure the number of '{EPOCHS}' in your configuration file."
+                f"Please configure the number of '{EPOCHS}' in your configuration file for {self.__class__.name}."
```
Comment (tabergma) [inline rasa/nlu/classifiers/diet_classifier.py]:
```suggestion
                f"Please configure the number of '{EPOCHS}' in your configuration file for '{self.__class__.name}'."
```

File: rasa/nlu/classifiers/diet_classifier.py [hunk 2]
```diff
@@ -647,8 +647,8 @@ def train(
     def _predict(self, message: Message) -> Optional[Dict[Text, tf.Tensor]]:
         if self.model is None:
             logger.debug(
-                "There is no trained model: component is either not trained or "
-                "didn't receive enough training data."
+                f"There is no trained model for {self.__class__.name}: component is either not trained or "
```
Comment (tabergma) [inline rasa/nlu/classifiers/diet_classifier.py]:
```suggestion
                 f"There is no trained model for '{self.__class__.name}': component is either not trained or "
```

File: rasa/nlu/classifiers/diet_classifier.py [hunk 3]
```diff
@@ -826,7 +826,7 @@ def load(
 
         if not model_dir or not meta.get("file"):
             logger.debug(
-                f"Failed to load model. "
+                f"Failed to load {cls.name}. "
```
Comment (tabergma) [inline rasa/nlu/classifiers/diet_classifier.py]:
```suggestion
                f"Failed to load '{cls.name}'. "
```

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @dakshvar22 will take a look at it as soon as possible ✨

**Pull request merged**